### PR TITLE
FIXED: Before this PR, nib2cib was using hard coded frame adjustment for CPBox

### PR DESCRIPTION
**Pull request by didierkorthoudt**
This PR uses #2831 to use a theme specific adjustment frame attribute instead of hard coded value. Default value corresponds to old hard coded value.

### CODE REVIEW

**Comment by didierkorthoudt** [comment]
#milestone=1.1

**Comment by didierkorthoudt** [comment]
#milestone=1.1

**Comment by daboe01** [comment]
-#new
+Theme
+#ready-to-commit

File: AppKit/CPBox.j [hunk 1]
```diff
@@ -90,7 +90,8 @@ CPBelowBottom = 6;
             @"inner-shadow-size": 6.0,
             @"inner-shadow-color": [CPNull null],
             @"content-margin": CGSizeMakeZero(),
-        };
+            @"nib2cib-adjustment-primary-frame": CGRectMake(4, -4, -8, -6)
```
Comment (Dogild) [inline AppKit/CPBox.j]: Why this is not called `nib2cib-adjustment-frame` as it is for other controls ?

**Comment by mrcarlberg** [comment]
I have two issues on this:
1. An answer on Dogild's question above
2. Are the default values correct? Looks like `y` is -4 and the removed code adds 4?

File: AppKit/CPBox.j (hunk 1 above)
Comment (didierkorthoudt) [reply AppKit/CPBox.j]: Well, in fact I wanted to show that this adjustment will only be applied on primary (and secondary) boxes (and not on all types of boxes)... This way, other types may use nib2cib-adjustment-frame the usual way…

**Comment by didierkorthoudt** [comment]
Regarding the +-4 question, I have to check but I think this is because when using nib2cib-adjustment-frame, vertical coordinates are not yet reversed.

File: Tools/nib2cib/NSBox.j [hunk 2]
```diff
@@ -80,6 +66,33 @@
     return self;
 }
 
+// Override
+
+- (CGRect)_nib2CibAdjustment
+{
+    if ((_boxType !== CPBoxSeparator) && ((_boxType === CPBoxPrimary) || (_boxType === CPBoxSecondary)))
```
Comment (mrcarlberg) [inline Tools/nib2cib/NSBox.j]: @didierkorthoudt One last thing..... I hope 😄.... Why does this `if` first test `_boxType !== CPBoxSeparator` and after that `(_boxType === CPBoxPrimary) || (_boxType === CPBoxSecondary)`. Is the first test really needed or am I missing something here?

Comment (didierkorthoudt) [reply Tools/nib2cib/NSBox.j]: You're absolutely right, I've just copied the old test and should have seen this… I'll adapt, thank you !

**Comment by didierkorthoudt** [comment]
I've simplified the test. Are we OK on this one ?

**Comment by mrcarlberg** [comment]
Merged! Thanks!

**Pull request merged**